### ISSUE TITLE
How to query sites other than originating site.

### ISSUE THREAD

**Issue by mwsaunders** [SharePoint/sp-dev-solutions#218]
#### Category
- [X] Question
- [ ] Bug
- [ ] Enhancement

Hi, I am still researching to find and answer, but figured I would get this question going.

I have the Modern Search Solution working good on a hub site. 
Using DepartmentId:{{PageContext.hubSiteId}} it is finding all items in all sites connected to the hub.

Now I am working on setting this up on a different site - one that is not associated to the hub being used above.  I would like to..
1) Query all sites associated to the hub, by url or GUID if necessary.
OR
2) Specify in the Query sites by Url or GUID if necessary.

I am unable to find the syntax in the Query to achieve this.

I am going to look into setting up a tenant level result source see where that gets me.
As I would envision this being setup on multiple other sites not associated to the main hub, if this works, seems a better solution.

Found this..
So this gets info from the Site of the hub.
Path:"https://devusesi.sharepoint.com/sites/MHConduit"
Any way to get all associated sites without specifying them separately?

Did I mention I am a novice as this.
Regards,

**Comment by wobba** [comment]
The hub id is the site id of the hub, so you could look up the site guid and use that on the other sites.

**Comment by mwsaunders** [comment]
Hi Wobba,  Yes I have the gui HubId.   
After some trial and error I found the correct syntax to be.
DepartmentId:{55f1a0a1-7555-4fc0-9416-9582ab40a3c1} OR Path:"https://xxxxxxxx.sharepoint.com/sites/MH-Private-R22a"
This finds items on the corporate hub and the non-associated regional site.

 I did happen across your site while researching.  I saw the picture and said, hmm that looks familiar.
Thanks for your help.
